### DISCUSSION TITLE
How to add App to "External App tests"

### DISCUSSION THREAD

**Discussion by WilkAndy** [idaholab/moose#16837] [Q&A Tools] [answered]
Hi,

@josebastiase and i have made a small MOOSE app called RHEA, https://github.com/josebastiase/RHEA.   It has two Materials and two tests associated with it.   How can we add it to the "External App tests" ?

a

**Comment by loganharbour** [comment, marked as answer]
Go ahead and shoot me an email, @WilkAndy, and we'll discuss the process of setting this up.

For anyone else curious about having your applications tested: Providing that your application is in active development and you're willing to contribute to keeping it up to date, we're glad to add it to testing!

Note that there are other requirements for approval, but those will be discussed one-on-one when making the request.
